config.json

```diff
@@ -13,9 +13,9 @@
             }
         },
         "datapipelines": {
-            "active": true,
+            "active": false,
              "features": {
-                 "show_stack_logs": "enabled"
+                 "show_stack_logs": "disabled"
```

Comment:
I am not sure we need to disable the visibility of the logs by default, what was the reasoning behind?